ios/sdk/sdk.xcodeproj/project.pbxproj

```diff
@@ -410,6 +428,7 @@
 			outputPaths = (
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/GTMOAuth2ViewTouch.nib",
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/GoogleSignIn.bundle",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/AntDesign.ttf",
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Entypo.ttf",
```

Comment:
Do we need the rest of the fonts here?

Reply:
That's automatically generated by CocoaPods because the fonts are listed as resources in the react-native-vector-icons podspec file. That's why we are deleting them with a script right after the CocoaPods bundle resources step.